modules/oe_content_sub_entity/modules/oe_content_sub_entity_author/oe_content_sub_entity_author.module

```diff
@@ -48,3 +49,28 @@ function oe_content_sub_entity_author_node_prepare_form(EntityInterface $entity,
   ]);
   $entity->set('oe_authors', [$corporate_body]);
 }
+
+/**
+ * Implements hook_entity_delete().
+ *
+ * Delete author sub entities referencing a node being deleted.
+ */
+function oe_content_sub_entity_author_entity_delete(EntityInterface $entity) {
```

Comment:
We need to test this code. If I remove this hook we won't have any failures on the pipeline.

Reply:
Adapted test coverage.